cmd/state-custom-installer/docker/centos-7/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM docker-registry.activestate.build/activestate/centos-7.6-builder:1.0.66 AS custom-installer-builder
```

Comment:
FROM docker-registry.activestate.build/activestate/centos-7.6-builder:1.0.67 AS custom-installer-builder

Reply:
Needs an increment.

Reply:
Probably make the centos-7 / Centos-8 a configurable option to the Docker build, as we're basically duplicating the code except for this line.